### PR TITLE
51 clustername sanitization leaves trailing slash which makes it an invalid hostname

### LINKED ISSUE #51: clusterName sanitization leaves trailing slash which makes it an invalid hostname
Discovered over in naps-dev/arkime#31, the clusterName that gets passed to k3d is originally sanitized by simple truncation to 32 chars to appease k3d's cluster name limitation. This can result in the clusterName being an invalid hostname (which is required since it's used in the names of nodes and perhaps other things) due to trailing slashes. This causes an error in Zarf Package tests used by everything in dco-core and its components.

### PR DESCRIPTION
**Pull request by timblaktu**
## Type of change

- [X] Bug fix (non-breaking change which fixes an issue)

# Description

Fixes #51 
Closes #51 

# How Has This Been Tested?

TLDR; This has been tested via the standard PR workflow on the source branch.

Originally, this was tested over in naps-dev/arkime#31 in order to unblock the validation of naps-dev/arkime#30. 

Since the fixes were actually in dco-core, I created this issue and corresponding branch and PR. The branch was taken from the same dco-core branch used in naps-dev/arkime#31, i.e. 30-update-to-latest-arkime-version, after updating it to main.

### CODE REVIEW

**Comment by timblaktu** [comment]
That last workflow finally got queued, but [it failed early authenticating to ECR](https://github.com/naps-dev/dco-core/actions/runs/5427680395/jobs/9871178887#step:11:24) - so I suspect root cause is same as what cause the previous failure (which was in `aws s3 cp` after an otherwise successful run :disappointed: )

I'll kick off another, adding some debug output in the ECR login stage..

File: test/dco_core_package_test.go [hunk 1]
```diff
@@ -4,6 +4,8 @@ import (
 	"context"
 	"net"
 	"os"
+  "regexp"
+  "strings"
```
Comment (bburky) [inline test/dco_core_package_test.go:8]:
```suggestion
	"regexp"
	"strings"
```

Comment (bburky) [reply test/dco_core_package_test.go:8]: Needs go fmt over the go code
